setup.py

```diff
@@ -291,7 +266,8 @@ def build_extension(self, ext):
     platforms=PLATFORMS,
     classifiers=CLASSIFIERS,
     package_dir=PACKAGE_DIR,
-    py_modules=PY_MODULES,
+    packages=["brotli"],
+    package_data={"brotli": ["*.py", "*.pyi", "py.typed"]},
```

Comment:
you shouldn't need to add *.py files to `package_data`, they are already included as part of being python modules contained inside one the listed `packages` directories ("brotli" in our case).

Reply:
Oh, you're right.
I will change it later.

I was somehow under the impression that one needs to specify all.

Reply:
yeah, package_data is only for pure data files associated with a given package, python modules inside listed packages are included automatically

Reply:
The whole package_data entry is not even necessary,
as the relevant files also get included via the manifest.in.

Reply:
Well, it depends. Adding them only to the MANIFEST.in will simply include them in the source distribution tarball; with package_data they would also be installed alongside the python files in the package directory , which I believe was what you wanted to do when you turned the single extension module into a package directory.
If you need to have the stub installed, then you need package_data.